pytket/tests/classical_test.py

```diff
@@ -458,6 +458,52 @@ def test_wasmfilehandler_repr_64() -> None:
     )
 
 
+def test_wasmfilehandler_repr_2() -> None:
```

Comment:
Looks like this test (and the following one) pass with the existing version, or am I missing something? Do they exhibit the bug?

Reply:
Ah OK, thanks!